README.md

```diff
@@ -1,2 +1,15 @@
-# collaborative-pixel-art
-A terrible idea that was shamelessly stolen from Reddit
+# Collaborative Pixel Art
+### A terrible idea that was shamelessly stolen from Reddit
```

Comment:
Probably doesn't need to be a header. It was fine as-is.

Reply:
It just looks nicer as a header imo